external-providers/java-external-provider/pkg/java_external_provider/provider.go

```diff
@@ -389,6 +390,7 @@ func (p *javaProvider) Init(ctx context.Context, log logr.Logger, config provide
 		"-Dosgi.checkConfiguration=true",
 		fmt.Sprintf("-Dosgi.sharedConfiguration.area=%s", sharedConfigPath),
 		"-Dosgi.sharedConfiguration.area.readOnly=true",
+		//"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:1044",
```

Comment:
Just curious about this commented line, is helpful for some local dev or lang-server debugging or something?

Reply:
Removed it from below because an empty `workspace` variable can cause issues